types/mochawesome/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// Type definitions for mochawesome 6.2
+// Project: https://github.com/adamgruber/mochawesome#readme
+// Definitions by: Chris Gilardi <https://github.com/Christop406>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface ReporterOptions {
+    quiet: boolean;
+    code: boolean;
+    'no-code': boolean;
+    html: boolean;
+    json: boolean;
+    consoleReporter: string;
```

Comment:
What about `reportFilename` (https://github.com/adamgruber/mochawesome/blob/master/src/config.js#L36)?

Reply:
> @Christop406 What about [#52889 (comment)](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52889#discussion_r631289760), [#52889 (comment)](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52889#discussion_r631292033), and [#52889 (comment)](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52889#discussion_r631296785)?

Sorry, I responded to one of your other comments. I didn't see the first two before pushing, but am working on it now. I did implement the `namespace` suggestion though.